### DISCUSSION TITLE
提交DDL脚本前端报错

### DISCUSSION THREAD

**Discussion by solon-liu** [hhyo/Archery#2386] [Q&A]
### 重现步骤

在SQL上线窗口，提交DDL脚本，点击SQL检测后前端页面提示报错

### 预期外的结果

![image](https://github.com/hhyo/Archery/assets/24712997/b626a2d5-7e30-48ba-a670-4cf0a867b89d)

### 日志文本

```shell
没有日志提示
```

### 版本

1.9.1

### 部署方式

Docker

**Comment by solon-liu** [comment]
DDL脚本手动执行是没问题的

**Comment by solon-liu** [comment]
目前我遇到的问题是所有DDL都会报错这个

**Comment by Grain-Yu** [comment]
升级到最新版本试试？

**Comment by woshiyanghai** [comment]
这好像是你的token过期了，目前archery 的token过期了，提示不规范，刷新页面重新登录应该就好了
